### PR TITLE
Feature/ge 1128

### PR DESCRIPTION
**Pull request by AltusBarry**
Change some translation marked strings, to make the po files more palatable.

### CODE REVIEW

File: authentication_service/templates/oidc_provider/authorize.html [hunk 1]
```diff
@@ -21,9 +21,9 @@ <h1>{% trans "Request for Permission" %}</h1>
 {% endblock %}
 
 {% block content %}
-    {% blocktrans with client_name=client.name %}
-        <p class="Intro">You recently logged in to <strong>{{ client_name }}</strong>, and they would like to access this information of you:</p>
-    {% endblocktrans %}
+    <p class="Intro">
+        {% blocktrans with client_name=client.name %}You recently logged in to {{ client_name }}. They would like to access this information of you:{% endblocktrans %}
```
Comment (DavidOberholzer) [inline authentication_service/templates/oidc_provider/authorize.html]: nit pick: Format of this one didnt change like the other ones above?

File: authentication_service/templates/two_factor/core/phone_register.html [hunk 2]
```diff
@@ -10,21 +10,23 @@ <h1>{% trans "Add Backup Phone" %}</h1>
...
-      <p>{% blocktrans %}You'll be adding a backup phone number to your
-        account. This number will be used if your primary method of
-        registration is not available.{% endblocktrans %}</p>
-  {% elif wizard.steps.current == 'validation' %}
-      <p>{% blocktrans %}We've sent a token to your phone number. Please
-        enter the token you've received.{% endblocktrans %}</p>
-  {% endif %}
+    {% if wizard.steps.current == 'setup' %}
+        <p>
+        {% blocktrans trimmed %}
```
Comment (DavidOberholzer) [inline authentication_service/templates/two_factor/core/phone_register.html]: nit pick: indentation

File: authentication_service/templates/oidc_provider/authorize.html (hunk 1 above)
Comment (AltusBarry) [reply authentication_service/templates/oidc_provider/authorize.html]: Yeah, left some of the shorter ones one line. Negates the need to add the "trimmed" arg to blocktrans.

File: authentication_service/templates/two_factor/core/phone_register.html (hunk 2 above)
Comment (AltusBarry) [reply authentication_service/templates/two_factor/core/phone_register.html]: Thanks.

**Pull request closed without merging**